apps/condo/domains/common/utils/serverSchema/index.js

```diff
@@ -234,7 +236,16 @@ const loadListByChunks = async ({
             throw new Error('Operation timed out')
         }
 
-        newChunk = await list.getAll(context, where, { sortBy, first: chunkSize, skip: skip })
+        // TODO INFRA-538 remove this condition once migration to new server utils to be done
+        const resolvedFields = !isNil(fields) ? fields : 'id'
+        if (get(list, 'hasFieldsParam') && isNil(fields)) {
+            // the case when we migrate model to use new server utils, but loadListByChunks use it in old way
+            console.trace('Provide required fields parameter for loadListByChunks util')
+        }
```

Comment:
Do you want the fields to be explicitly passed?
Isn't it enough that there is a default value ("id")?

Reply:
Yes, 99.9% usages of loadListByChunks - get model data, not only ids. There are another case: model was migrated to the new codegen utils, but loadListByChunks are calling in old way. So I would like to notify developer about such case - to trigger him to rewrite code and provide required "fields" param.